### PR TITLE
contribute-to-open-food-facts-by-playing.html

### PR DESCRIPTION
**Pull request by teolemon**
hunger-games-help-english
contribute-to-open-food-facts-by-playing.html

### CODE REVIEW

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<!-- no side column -->
+
+<h1>Contribute to Open Food Facts by playing!</h1>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:3]:
```suggestion
<h1>Play and contribute to Open Food Facts!</h1>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+<h2 class="subheader">The contribution to Open Food Facts becomes child's play thanks to this game imagined and created by the voluntary developers of Open Food Facts.</h2>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:5]:
```suggestion
<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteers developers.</h2>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<!-- no side column -->
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteers developers.</h2>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:5]:
```suggestion
<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteer developers.</h2>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:10]:
```suggestion
<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+<p><strong>This game allows you to modify product sheets simply, quickly and in a fun way</strong>. Enough to make the contribution to an Open Source project addictive.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:11]:
```suggestion
<p><strong>This game allows you to modify products easily, quickly and in a fun way</strong>. Enough to make contributing to an Open Source project addictive.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions people to choose better foods and public health research teams for their studies.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:12]:
```suggestion
<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods as well as public health research teams for their studies.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+<p class="callout panel">To play it is simple, you just have to connect to Open Food Facts and then click on <a href="https://hunger.openfoodfacts.org/">this link</a>.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:14]:
```suggestion
<p class="callout panel">To start playing, you just have to log in to your Open Food Facts account and then click on <a href="https://hunger.openfoodfacts.org/">this link</a>.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+<p><strong>Want to know more before you get started?</strong>Find out more about the game below!</p>
+
+<h3>How does this game work?</h3>
+
+<p>The game randomly loads a base product that needs to be completed. The interface is simple, it allows you to interact by answering a question about an attribute of the product by "Yes / No / Skip" and get more product information if needed.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:20]:
```suggestion
<p>The game randomly loads a product from Open Food Facts that needs to be completed. The interface is simple, allowing you to interact by answering a question about an attribute of the product by "Yes, Skip or No" and get more product information if needed.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+<p>The interface is divided into 3 parts:</p>
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically go to the next product.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:26]:
```suggestion
<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html (hunk 4 above)
Comment (GendelfLugansk) [reply lang/en/texts/contribute-to-open-food-facts-by-playing.html:10]: https://world-en.openfoodfacts.org/help-complete-products redirects me to https://world.openfoodfacts.org/help-complete-products

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+</div>
+
+<p>If you have any questions or comments, please don't hesitate to contact us on the Slack or on <a href="https://www.facebook.com/groups/openfoodfacts.fr/">the Facebook group</a>.</p>
+
+<h2>Thank you!</h2>
+
+<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can
+change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
+
+<p>So your contributions help millions of people who use our mobile app and website to choose better food,
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:71]:
```suggestion
<p>So your contributions will help millions of people who use our mobile app and website to choose better food,
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+</ul>
+
+</div>
+
+<p>If you have any questions or comments, please don't hesitate to contact us on the Slack or on <a href="https://www.facebook.com/groups/openfoodfacts.fr/">the Facebook group</a>.</p>
+
+<h2>Thank you!</h2>
+
+<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can
+change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:69]:
```suggestion
<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+<p><strong>When you are not 100% sure of your answer you must answer "Skip"!</strong>Above all, do not answer "Yes" or "No" if you are not sure of your choice.
+Another person will take care of qualifying this data later! This game is above all collaborative and aims to contribute efficiently and qualitatively to Open Food Facts data.</p>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">2. Access information</h3>
+<p>If the answer is not visible on the product image, this part of the screen allows you to view the product information. You can click on:</p>
+<ul>
+<li>Images to zoom,</li>
+<li>The "view" button: provides access to the product file,</li>
+<li>The "edit" button: gives access to the file's editing interface.</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:51]:
```suggestion
<li>The "View" button: provides access to the product page on the Web version of Open Food Facts,</li>
<li>The "Edit" button: lets you edit the product on Open Food Facts,.</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+
+<p>The game randomly loads a product from Open Food Facts that needs to be completed. The interface is simple, allowing you to interact by answering a question about an attribute of the product by "Yes, Skip or No" and get more product information if needed.</p>
+
+<p>The interface is divided into 3 parts:</p>
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of data you wish to complete:</p>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:28]:
```suggestion
<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of questions you wish to answer:</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+<ul>
+<li><strong>label</strong>: relating to the "label" field (eg. certification, product origin, etc.)</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:30]:
```suggestion
<li><strong>label</strong>: Questions relating to the "label" field (eg. certification, product origin, etc.)</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+<li><strong>category</strong>: relating to the "category" field (eg drinks, sweets, etc.)</li>
+<li><strong>brand</strong>: relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
+<li><strong>product_weight</strong>: relating to the weight of the product (eg 100 g, 1L, etc.)</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:33]:
```suggestion
<li><strong>category</strong>: Questions relating to the "category" field (eg drinks, sweets, etc.)</li>
<li><strong>brand</strong>: Questions relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
<li><strong>product_weight</strong>: Questions relating to the weight of the product (eg 100 g, 1L, etc.)</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+
+<h1>Play and contribute to Open Food Facts!</h1>
+
+<h2 class="subheader">Contributing to Open Food Facts becomes child's play thanks to this game imagined and created by some of our volunteer developers.</h2>
+
+<div class="button_div unselectbuttondiv_front_en">
+	<a href="https://hunger.openfoodfacts.org/" class="button">Play now</button></a>
+<div>
+<p>We thank the many volunteers who have already <a href="https://world-en.openfoodfacts.org/help-complete-products">helped complete products</a> and invite you to discover this game.</p>
+<p><strong>This game allows you to modify products easily, quickly and in a fun way</strong>. Enough to make contributing to an Open Source project addictive.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:11]: "Enough to make contributing to an Open Source project addictive."  --> the "addictive" part is not really positive in my mind. And the sentence without a verb is odd.

Maybe:

"This game allows you to contribute to an open source project that benefits everyone in an easy, quick and fun way!"

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods as well as public health research teams for their studies.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:12]: the "better foods as well as public health research teams" is hard to follow, I think it will be better with an explicit comma to separate the 2 parts:

```suggestion
<p class="callout" style="font-size:18px;padding:1%;border-top:dashed 1px #000;border-bottom:dashed 1px #000;">The information you add will be used by millions of people to choose better foods, and by public health research teams for their studies.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+
+<p class="callout panel">To start playing, you just have to log in to your Open Food Facts account and then click on <a href="https://hunger.openfoodfacts.org/">this link</a>.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:14]: We can suggest to create an OFF account (it makes the mission more friendly to newcomers who may not have an account yet).

```suggestion
<p class="callout panel">To start playing, create an Open Food Facts account, log-in, and click on <a href="https://hunger.openfoodfacts.org/">this link</a>.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 19]
```diff
@@ -0,0 +1,73 @@
...
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of questions you wish to answer:</p>
+<ul>
+<li><strong>label</strong>: Questions relating to the "label" field (eg. certification, product origin, etc.)</li>
+<li><strong>category</strong>: Questions relating to the "category" field (eg drinks, sweets, etc.)</li>
+<li><strong>brand</strong>: Questions relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:32]: We should remove the brands examples, they make the page not i18n friendly.

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 20]
```diff
@@ -0,0 +1,73 @@
...
+
+<p>The interface is divided into 3 parts:</p>
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of questions you wish to answer:</p>
+<ul>
+<li><strong>label</strong>: Questions relating to the "label" field (eg. certification, product origin, etc.)</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:30]: "Question relating" is unnecessarily vague --> "Whether the brand has a specific label such as "Organic" or "Fair trade"

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 21]
```diff
@@ -0,0 +1,73 @@
...
+<li><strong>category</strong>: Questions relating to the "category" field (eg drinks, sweets, etc.)</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:31]:
```suggestion
<li><strong>category</strong>: Whether the product belongs to a specific category such as "Yogurts" or "Chocolate cookies"</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 22]
```diff
@@ -0,0 +1,73 @@
...
+<li><strong>brand</strong>: Questions relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
+<li><strong>product_weight</strong>: Questions relating to the weight of the product (eg 100 g, 1L, etc.)</li>
+</ul>
+<p>Below we find:</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:35]:
```suggestion
<p>You can see below:</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 23]
```diff
@@ -0,0 +1,73 @@
...
+<ol>
+<li>A search field, allowing you to specify which brand, label, category you want to work on,</li>
+<li>The question to be answered in English. In English, this gives: "Does this product correspond to the indicated value?"</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:38]: Maybe just:

"The question to be answered".

The "English" part is confusing, let's assume people who will use Hunger Game know some English (otherwise they won't use it anyhow).

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 24]
```diff
@@ -0,0 +1,73 @@
...
+<li>Product image,</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:39]: a photo of the product

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 25]
```diff
@@ -0,0 +1,73 @@
...
+<li>3 clickable buttons to answer the question.</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:40]: 3 buttons.  (clickable is not needed)

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 26]
```diff
@@ -0,0 +1,73 @@
...
+</ol>
+
+<p><strong>When you are not 100% sure of your answer you must answer "Skip"!</strong>Above all, do not answer "Yes" or "No" if you are not sure of your choice.
+Another person will take care of qualifying this data later! This game is above all collaborative and aims to contribute efficiently and qualitatively to Open Food Facts data.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:44]:
```suggestion
Another person will take care of answering this question later. This game is above all collaborative and aims to contribute efficiently and qualitatively to Open Food Facts data.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 27]
```diff
@@ -0,0 +1,73 @@
...
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">2. Access information</h3>
+<p>If the answer is not visible on the product image, this part of the screen allows you to view the product information. You can click on:</p>
+<ul>
+<li>Images to zoom,</li>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:49]:
```suggestion
<li>Product photos you can zoom on,</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 28]
```diff
@@ -0,0 +1,73 @@
...
+</div>
+
+<p>If you have any questions or comments, please don't hesitate to contact us on the Slack or on <a href="https://www.facebook.com/groups/openfoodfacts.fr/">the Facebook group</a>.</p>
+
+<h2>Thank you!</h2>
+
+<p>Open Food Facts is a non-profit project that could not exist without your help. Data on the foods we eat have many uses and can change the lives of many people. This is why all the data we collect is freely accessible to everyone, in open data, so that we can build a common good that benefits everyone.</p>
+
+<p>So your contributions will help millions of people who use our mobile app and website to choose better food,
+but also public health researchers who use our data for their studies.</p>
```
Comment (stephanegigandet) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:71]:
```suggestion
as well as public health researchers who use our data for their studies.</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html (hunk 16 above)
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:11]:
```suggestion
<p><strong>This game allows you to contribute to an open source project that benefits everyone in an easy, quick and fun way!</p>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 29]
```diff
@@ -0,0 +1,73 @@
...
+
+<p>The interface is divided into 3 parts:</p>
+<div><img style="max-height:400px" src="/images/misc/hunger-game-0.png"></div><br>
+
+<h3 style="background-color:#00d400;padding:1%;color:#fff;font-weight:bold;">1. The question</h3>
+<p>To answer the question, just click on one of the 3 options (Yes/No/Skip) and the game will automatically move onto the next product.</p>
+
+<p>At the top of the screen are 4 buttons (filters) allowing you to select the type of questions you wish to answer:</p>
+<ul>
+<li><strong>label</strong>: Questions relating to the "label" field (eg. certification, product origin, etc.)</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:30]:
```suggestion
<li><strong>label</strong>: Whether the brand has a specific label such as "Organic" or "Fair trade"</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 30]
```diff
@@ -0,0 +1,73 @@
...
+<li><strong>category</strong>: Questions relating to the "category" field (eg drinks, sweets, etc.)</li>
+<li><strong>brand</strong>: Questions relating to the brand of the product (eg Sodébo, Charal, Alter Eco, etc.)</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:32]:
```suggestion
<li><strong>brand</strong>: Questions relating to the brand of the product</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 31]
```diff
@@ -0,0 +1,73 @@
...
+<li><strong>product_weight</strong>: Questions relating to the weight of the product (eg 100 g, 1L, etc.)</li>
+</ul>
+<p>Below we find:</p>
+<ol>
+<li>A search field, allowing you to specify which brand, label, category you want to work on,</li>
+<li>The question to be answered in English. In English, this gives: "Does this product correspond to the indicated value?"</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:38]:
```suggestion
<li>The question to be answered, such as "Does this product correspond to the indicated value?"</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 32]
```diff
@@ -0,0 +1,73 @@
...
+<li>Product image,</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:39]:
```suggestion
<li>a photo of the product,</li>
```

File: lang/en/texts/contribute-to-open-food-facts-by-playing.html [hunk 33]
```diff
@@ -0,0 +1,73 @@
...
+<li>3 clickable buttons to answer the question.</li>
```
Comment (teolemon) [inline lang/en/texts/contribute-to-open-food-facts-by-playing.html:40]:
```suggestion
<li>3 buttons to answer the question.</li>
```

**Comment by teolemon** [comment]
Suggestions applied @stephanegigandet
